.github/workflows/ci.yml

```diff
@@ -78,6 +78,36 @@ jobs:
         run: yarn test-unit
       - name: Integration Tests
         run: yarn test-integration
+  test-ibmi:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [12, 16]
+    name: IBM i (Node ${{ matrix.node-version }})
+    runs-on: ubuntu-latest
+    needs: test-typings
+    env:
+      DIALECT: ibmi
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install --frozen-lockfile --ignore-engines
+      - name: Install unixodbc
```

Comment:
I believe this can be removed, we should be able to do the same as with [test-snowflake](https://github.com/sequelize/sequelize/blob/65a6fe904da3aea896734c41f6d61b9b7d668497/.github/workflows/ci.yml#L274-L291) iirc